projects/distributed-tracing/src/pages/trace-detail/logs/trace-logs.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { LogEventsTableViewType } from '../../../shared/components/log-events/log-events-table.component';
+import { LogEvent } from '../../../shared/dashboard/widgets/waterfall/waterfall/waterfall-chart';
+import { TraceDetailService } from './../trace-detail.service';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <ng-container *ngIf="this.logEvents$ | async as logEvents">
+      <ht-log-events-table
+        [logEvents]="logEvents"
+        logEventsTableViewType="${LogEventsTableViewType.Detailed}"
+      ></ht-log-events-table>
+    </ng-container>
+  `
+})
+export class TraceLogsComponent {
```

Comment:
So this is not a dash page. It works. But I am not sure if we should be diverge from the dashboard pattern. 

@aaron-steinfeld  thoughts?

Reply:
IMO, we should try to reign dashboards back in to pages that look like dashboards. If we later need to embed this as a widget in another page, we could widgetize it for sharing at that point - but no need to build these custom single-page, single-purpose widgets.